packages/shared/src/lib/core/nfts/actions/addOrUpdateNftInAllAccountNfts.ts

```diff
@@ -17,3 +17,19 @@ export function addOrUpdateNftInAllAccountNfts(accountIndex: number, ...newNfts:
         return state
     })
 }
+
+export function addOrUpdateAllAccountNfts(...newNfts: INft[]): void {
```

Comment:
updateAllAccountNfts for this one

updateAllAccountNftsForAccount for the one above